azure-specialized-workloads/avd/recommendations.yaml

```diff
@@ -1,154 +1,181 @@
-  potentialBenefits: Enhances AVD reliability
+    Use Service Health to stay informed about the health of the Azure services and regions that you use to insure their availability. Set up Service Health alerts so that you stay aware of service issues, planned maintenance, or other changes that might affect your Azure Virtual Desktop resources. Use Resource Health to monitor your VMs and storage solutions.
+  potentialBenefits: Enhanced AVD error tracking and resolution
   pgVerified: true
   publishedToLearn: false
   publishedToAdvisor: false
-  automationAvailable: no
+  automationAvailable: ARG
   tags: null
   learnMoreLink:
     - name: Learn More
-      url: "https://learn.microsoft.com/en-us/azure/architecture/example-scenario/wvd/windows-virtual-desktop#azure-virtual-desktop-limitations"
-    - name: Private Link
-      url: "https://learn.microsoft.com/en-us/azure/well-architected/azure-virtual-desktop/networking#private-endpoints-private-link"
+      url: "https://learn.microsoft.com/azure/well-architected/azure-virtual-desktop/monitoring#resource-health"
 
-- description: Deploy Domain Controllers and DNS Servers in Azure Virtual Network Across Availability Zones
-  aprlGuid: 99bf5c94-aa68-4bb3-8b7f-45d1c5f09b5d
+- description: Configure AVD Insights workbook
+  aprlGuid: 0cf72d91-644d-4591-9bb7-84ba3f705a41
   recommendationTypeId: null
-  recommendationControl: High Availability
+  recommendationControl: Monitoring and Alerting
   recommendationImpact: High
   recommendationResourceType: n/a
   recommendationMetadataState: Active
   longDescription: |
-    When using an AD DS identity solution with AVD, it is recommended to deploy domain controllers and DNS servers on Azure virtual machines across availability zones. This improves the environment's reliability by removing a dependency on an on-premises service and improves performance by creating a shorter path for user authentication.
-    This recommendation is not relevant when you are utilizing Microsoft Entra as the identity provider.
-  potentialBenefits: Enhanced reliability and performance
+    AVD Insights is an Azure Workbook template provided by the AVD product team. It is highly recommended in order to monitor and troubleshoot AVD workloads across metrics, logs, events, and more. Both Production and DR workloads should be enabled with AVD Insights.
+  potentialBenefits: Enhanced AVD monitoring and troubleshooting
   pgVerified: true
   publishedToLearn: false
   publishedToAdvisor: false
   automationAvailable: no
   tags: null
   learnMoreLink:
     - name: Learn More
-      url: "https://learn.microsoft.com/en-us/azure/architecture/example-scenario/identity/adds-extend-domain#reliability"
+      url: "https://learn.microsoft.com/azure/virtual-desktop/insights?tabs=monitor"
```

Comment:
Confirmed with Dany that he intends for these to be different recommendations which is why they have new aprlGuids.